Developed in-house, not fed back.
Solaris-specific: MysqlTestRun customization.
=== modified file 'mysql-test/mysql-test-run.pl'
--- a/mysql-test/mysql-test-run.pl
+++ b/mysql-test/mysql-test-run.pl
@@ -100,6 +100,7 @@
 use IO::Socket::INET;
 use IO::Select;
 
+push @INC, ".";
 require "lib/mtr_process.pl";
 require "lib/mtr_io.pl";
 require "lib/mtr_gcov.pl";
@@ -2178,6 +2179,7 @@
 
   my $lib_plugin=
     mtr_file_exists(vs_config_dirs($location,$plugin_filename),
+                    "$basedir/lib/64/plugin/".$plugin_filename,
                     "$basedir/lib/plugin/".$plugin_filename,
                     "$basedir/lib64/plugin/".$plugin_filename,
                     "$basedir/$location/.libs/".$plugin_filename,

